pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
+
+- Resize all jpg images in the folder to 50% of their initial size:
+
+`mogrify -resize 50% *.jpg`
+
+- Resize all images starting with DSC to 800x600:
+
+`mogrify -resize 800x600 DSC*`
+
+- Convert all png images in the folder to jpg:
+
+`mogrify -format jpg *.png`
+
+- Halve the saturation for all files in the folder:
```

Comment:
[imagemagick supports practically any image file you can think of](https://hastebin.com/irawibipit).